test/integration/chai/chai-policy-tests.cxx

```diff
@@ -13,14 +13,25 @@ static_assert(RAJA::detail::get_space<RAJA::omp_parallel_for_exec>::value == cha
 
 #if defined(RAJA_ENABLE_CUDA)
 static_assert(RAJA::detail::get_space<RAJA::cuda_exec<128> >::value == chai::GPU, "");
+#endif
+
+#if defined(RAJA_ENABLE_CUDA)
 static_assert(RAJA::detail::get_space<RAJA::IndexSet::ExecPolicy<RAJA::seq_segit, RAJA::cuda_exec<128> > >::value == chai::GPU, "");
```

Comment:
This is just to test that a CUDA policy maps to the right chai space. If you look at the `get_space` it will work for any block size.